about/triage-meeting.md

```diff
@@ -10,36 +10,31 @@ The triage meeting takes place on [Zulip](chat-platform.md). Look for a topic li
 "#54818 weekly meeting YYYY-MM-DD" or "weekly meeting YYYY-MM-DD" (with the proper date, of course).
 
 ## Working group check-in
-This section contains the scheduled check-ins for working groups:
-
-- **2019-03-21:** [wg-nll], [wg-traits]
-- **2019-03-28:** [wg-parallel-rustc], [wg-pgo]
-- **2019-04-04:** [wg-rls-2.0], [wg-meta]
-- **2019-04-11:** [wg-mir-opt], [wg-pipelining]
-- **2019-04-18:** [wg-llvm], [wg-async-await],
-- **2019-04-25:** [wg-rfc-2229], [wg-self-profile]
-- **2019-05-02:** [wg-meta], [wg-rls-2.0]
-- **2019-05-09:** [wg-nll]
-
-Looking for a meeting that isn't listed above? Make a PR and extend the list to include that
-meeting.
-
-### Previous check-ins
-This section lists check-ins from triage meetings before the check-in schedule was introduced:
-
-- **2019-03-14:** [wg-async-await], [wg-llvm]
-- **2019-03-07:** [wg-self-profile], [wg-rfc-2229]
-- **2019-02-28:** [wg-rls-2.0], [wg-meta]
-
-[wg-rls-2.0]: ../working-groups/rls-2.0
-[wg-meta]: ../working-groups/meta
-[wg-self-profile]: ../working-groups/self-profile
-[wg-rfc-2229]: ../working-groups/rfc-2229
-[wg-async-await]: ../working-groups/async-await
-[wg-llvm]: ../working-groups/llvm
-[wg-nll]: ../working-groups/nll
-[wg-traits]: ../working-groups/traits
-[wg-parallel-rustc]: ../working-groups/parallel-rustc
-[wg-pgo]: ../working-groups/pgo
-[wg-mir-opt]: ../working-groups/mir-opt
-[wg-pipelining]: ../working-groups/pipelining
+Each week, two working groups check-in with the compiler team at the weekly triage meeting. To
+determine which groups need to check-in without maintaining a schedule that will need regularly
+updated or completely changed when new working groups are added, there exists a simple-ish formula
+to work out which groups should check-in:
+
+```
+index = (week * 2) % EVEN(num_working_groups)
+```
+
+..where `week` is the week number (0-indexed) and `EVEN(..)` rounds to the next positive number.
```

Comment:
I believe you mean "rounds to next **even** number"

Reply:
Fixed, thanks!